src/services/telegram/app.py

```diff
@@ -0,0 +1,57 @@
+import logging
+import cherrypy
+
+from common.WIOTRestApp import *
+from common.SettingsManager import *
+from common.SettingsNode import *
+from common.RESTBase import RESTBase
+from common.CatalogRequest import CatalogRequest
+from TelMan import MyBot
+
+class TelegramAdaptorAPI(RESTBase):
+
+    def __init__(self, upperRESTSrvcApp, settings: SettingsNode, telegramkey) -> None:
+        super().__init__(upperRESTSrvcApp, 0)
+        self._catreq = CatalogRequest(self.logger, settings)
+        self._bot = MyBot(telegramkey)
+
+    @cherrypy.tools.json_out()
+    def GET(self, *path, **args):
+
+        if path[0] == "sendMessage":
+            print("branch taken")
```

Comment:
```suggestion
            self.logger.debug("branch taken")
```

Reply:
I directly deleted the line of code. I used it for debug,now is useless. Anyway, logger.debug wasn't printing nothing in the logs , suggest logger.info

Reply:
This happens because the logger is set up in such a way to print only logs higher than debug. To change it you have to edit the line that initialize it

```python
class App(WIOTRestApp):
    def __init__(self) -> None:

        super().__init__(log_stdout_level=logging.INFO)
```

You've to change `logging.INFO` with `logging.DEBUG`